trademapper/js/trademapper.js

```diff
@@ -136,15 +136,46 @@ function($, d3, arrows, csv, filterform, mapper, route, yearslider,
 			.attr("id", "mapcanvas")
 			.attr("class", "map-svg flow")
 			.attr("viewBox", "0 0 "+this.config.width+" "+this.config.height);
+
+		var style = this.tmsvg.append("style");
+
+		style.text(
+			"/* <![CDATA[ */" +
+			".country { fill: " + config.colours["COUNTRY"] + "; }" +
```

Comment:
Maybe put this css in a template file? 

I guess the file data can be read with require.js. For the templating part maybe can use ES6 Template strings (they're supported from Edge 12 onwards, so no IE support) eg. https://stackoverflow.com/questions/25239704/replace-keys-in-template-string-with-object-properties

Or maybe without template strings:

```
template.replace(/\{(.*?)\}/g, function (e) {return replacements[e.slice(1, -1)];});
```

Reply:
<- obviously the single { in my code above won't work with css, we should either use two {{, or a different separator :)

Reply:
OK, good idea.